.eslintrc

```diff
@@ -44,6 +44,7 @@
     "react/react-in-jsx-scope": "off",
     "react/require-default-props": 0,
     "semi": 1,
+    "no-trailing-spaces": 0,
```

Comment:
Also what reason are there for allowing trailing spaces? Could potentially add noise to a diff. Couldn't it at least be a 1: warning?

Reply:
Totally agree. I'll set to 1 :)